dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -95,7 +98,7 @@ def get_torchbench_tpu_config(
       ),
       "rm -rf ~/xla/benchmarks/output/metric_report.jsonl",
       "python ~/xla/benchmarks/result_analyzer.py --output-format=jsonl",
-      f"gsutil cp {local_output_location} {gcs_location}",
+      f"gsutil cp {local_output_location} ${{gcs_location}}",
```

Comment:
Can you clarify the double brace `${{` syntax here? Looking at the code, I think this will format to something like `${gs://.../torchbench_config/metric_report_tpu.jsonl}`, which would not be a valid shell variable

Reply:
This will be formatted to string like `gsutil cp /tmp/metric ${gcs_location}`, where it will use the env `${gcs_location}`.

Reply:
Oh, I understand now. You're using an f-string for `{local_output_location}`, and `{{` escapes to `{` so you can reference the shell variable `${gcs_location}`.

Can you find another to format this string? Maybe just put `$gcs_location` to eliminate ambiguity around the braces

Reply:
Right, `$gcs_location` should be simpler.